src/redux/utilities.js

```diff
@@ -1,4 +1,4 @@
-export function formatContentType(contentType) {
+export default function formatContentType(contentType) {
```

Comment:
why did this get promoted to a default export?

Reply:
Good catch - happened as a result of the rebase from master - this file was in conflict - checking in change to resolve